src/lib/mina/fetch.ts

```diff
@@ -710,7 +710,15 @@ async function fetchActions(
     graphqlEndpoint,
     networkConfig.archiveFallbackEndpoints
   );
-  if (error) throw Error(error.statusText);
+  if (error) {
+    // retry once without querying transaction info
+    [response, error] = await makeGraphqlRequest<ActionQueryResponse>(
```

Comment:
Rather than retry the query, perhaps we can set the archive node API version in the Mina.Network settings somewhere.  Then we could make a different query based on which version we believe we're talking to.

For this specific example, after we get Minascan to update their version, the new query will work in nearly every case, but we ought to prepare for the case that different versions of the API could be running.